style.css

```diff
@@ -0,0 +1,65 @@
+figure {
+  display: block;
+  margin-block-start: 1em;
+  margin-block-end: 1em;
+  margin-inline-start: 40px;
+  margin-inline-end: 40px;
+}
+
+figcaption {
+  margin-top: 6px;
+  display: block;
+  text-align: center;
+}
+
+img {
+  display: block;
+}
+
+#main-header {
+  margin: auto;
+  margin-bottom: 6px;
+  padding: 15px;
+  font-family: 'Labrada', serif;
+  font-size: larger;
+  background-color: rgb(228, 253, 225);
+  text-align: center;
+}
+
+#image {
+  max-width: 100%;
+  height: auto;
+  align-self: center;
+}
+
+#tribute-info {
```

Comment:
The use of margin padding and font variations would make it more readable

Reply:
Great job👏 on making a polite request for a change! it would be helpful to provide some context on why the change you're requesting is necessary. This can help the student understand the reasoning behind the change and prevent them from making similar mistakes in the future. By explaining the importance of the change, the student can also learn more effectively and improve their coding skills. Keep up the good work!👍